src/test/java/com/google/devtools/build/lib/actions/DigestUtilsTest.java

```diff
@@ -213,6 +213,50 @@ public void testCache() throws Exception {
     assertThat(digest3).isEqualTo(digest1);
   }
 
+  @Test
+  public void testCacheWithSymlinks() throws Exception {
```

Comment:
nit: Test names should include the condition being tested and the expected outcome.

So I'd suggest something like `testCacheWithSymlinks_cacheHit`.

Reply:
Thanks for looking at this @joeleba . I'll change this right away.